modules/ide/xsk-ide-xsaccess/src/main/resources/META-INF/dirigible/xsk-ide-xsaccess/editor/editor.js

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) 2022 SAP SE or an SAP affiliate company and XSK contributors
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and XSK contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+angular.module('page', [])
+  .controller('PageController', function ($scope) {
+
+    let messageHub = new FramesMessageHub();
+    let contents;
+
+    function getResource(thePath) {
+      let xhr = new XMLHttpRequest();
+      xhr.open('GET', thePath, false);
+      xhr.send();
+      if (xhr.status === 200) {
+        return xhr.responseText;
+      }
+    }
+
+    function loadContents(xsaccessfile) {
+      if (!xsaccessfile) {
+        console.error('XSACCESS file parameter is not present in the URL');
```

Comment:
Shouldn't we display an error to the user or something?

Reply:
I don't think this `console.error` is really useful though. Remove this unless you think it's useful somehow

Reply:
will be removed.